### ISSUE TITLE
Blurred

### ISSUE THREAD

**Issue by Bao2080** [ashish-chopra/ngx-gauge#156]
<img width="293" alt="Screenshot 2024-10-15 at 4 17 38 PM" src="https://github.com/user-attachments/assets/d1dc4f33-13b4-460d-b103-527867849128">

Angular 18, Standalone.
html code:
```
<foreignObject x="241.73" y="430.1" width="100" height="100" >
  <div class="ngx-ahu-g4pf">
    <ngx-gauge
      style="background-color: white;"
      [value]="numberGauge" 
      [size]="80" 
      [label]="''"
      [append]="'Pa'" 
      [margin]="0"
      [min]="0" 
      [max]="100" 
      [thick]="5" 
      [type]="'arch'"
      [thresholds]="thresholds">
    </ngx-gauge>
  </div>
</foreignObject>
```

I have tried resizing. But the lines are still blurry whether they are inside the SVG or not.
How can I solve this situation?

**Comment by clementolive** [comment]
Hi, have you tried running it on another browser ? Which one did you use ?

**Comment by Bao2080** [comment]
> Hi, have you tried running it on another browser ? Which one did you use ?

yes, i tried on Google Chrome, Microsoft Edge, Android Phone(Samsung' and Huawei' mobile phone browser). The same problem will occur.

**Comment by DrakeAnglin** [comment]
@Bao2080 You need to play with the size option. your `size` is set to 80, but it looks like the gauge is zoomed in. if you increase the `size`, that basically increases the resolution. you will also have to increase the `thick` as that is based off size to get the dimension that your looking for.
